### PR TITLE
Cleanup admin controller file

### PR DESCRIPTION
**Pull request by Martii**
* Don't use `options` for JSON and plain text outputs instead use module helper ... faster/cleaner... this is also a Unit Test in this area as well as #249 checking
* Added watchpoint for #200 with `encodeURI`
* ~~Only time I've encounter `nil` helper function **so far**... added notation on inconsistency and since it's the only helper function used at this time in this file condensed the `require`~~
* ~~Added watchpoint for an encountered special render op... first I've seen of these so far... I know what the output looks like but unable to find a matching view for it other than it's visual reference in Admin Tools~~

**NOTES**
* ~~Questions to follow~~
* ~~One TODO to finish depending on answers~~

Originally applies to #262 ... not part of https://github.com/OpenUserJs/OpenUserJS.org/issues/262#issuecomment-5759212

### CODE REVIEW

File: controllers/admin.js [hunk 1]
```diff
@@ -86,7 +102,7 @@ exports.userAdmin = function (aReq, aRes, aNext) {
       });
     });
 
-    aRes.render('userAdmin', options);
+    aRes.render('userAdmin', options); // NOTE: Watchpoint
```
Comment (Martii) [inline controllers/admin.js]: @sizzlemctwizzle Cc: @Zren 

Do either one of you know where this physical view is? The docs for *express* [res.render](http://expressjs.com/4x/api.html#res.render) *(and related [app.render](http://expressjs.com/4x/api.html#app.render))* don't seem to cover this.

File: controllers/admin.js [hunk 2]
```diff
@@ -58,10 +70,14 @@ function getOAuthStrategies(aStored) {
 
 // Allow admins to set user roles and delete users
 exports.userAdmin = function (aReq, aRes, aNext) {
-  var options = nil();
+  //
+  var options = nil(); // NOTE: Inconsistent usage of object creation here
```
Comment (Martii) [inline controllers/admin.js]: @sizzlemctwizzle Cc: @Zren 

Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.

File: controllers/admin.js [hunk 3]
```diff
@@ -270,7 +328,7 @@ exports.adminApiKeysPage = function (aReq, aRes, aNext) {
   // strategyListQuery
   tasks.push(function (aCallback) {
     Strategy.find({}, function (aErr, aStrats) {
-      var stored = nil();
+      var stored = nil();  // NOTE: Inconsistent usage of object creation here
```
Comment (Martii) [inline controllers/admin.js]: @sizzlemctwizzle Cc: @Zren 

Again... Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.

File: controllers/admin.js [hunk 4]
```diff
@@ -381,7 +428,7 @@ exports.apiAdminUpdate = function (aReq, aRes, aNext) {
   });
 
   Strategy.find({}, function (aErr, aStrats) {
-    var stored = nil();
+    var stored = nil(); // NOTE: Inconsistent usage of object creation here
```
Comment (Martii) [inline controllers/admin.js]: @sizzlemctwizzle Cc: @Zren 

Again... Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.

File: controllers/admin.js (hunk 1 above)
Comment (Zren) [inline controllers/admin.js]: * https://github.com/Martii/OpenUserJS.org/commit/390b095ed837760214239ddfb5897e078ea4b348#diff-d8302114d2ebccc9c7f87ed04c5c78d4R1

* https://github.com/Martii/OpenUserJS.org/commits/master/views/userAdmin.html

Comment (Martii) [inline controllers/admin.js]: Ahhh so dead code from a different code migration... Thank you!

**Pull request merged**